{"name": "test_not_found_digit[0]", "status": "passed", "description": "\n            :test: тестовый id для подстановки в url\n        ", "parameters": [{"name": "test", "value": "0"}], "start": 1533894025575, "stop": 1533894026265, "uuid": "c2e97e8b-768b-439c-87fe-bf2b28728c83", "historyId": "058c694efaf65bda9a54aab8e70e2234", "fullName": "tests.get.test_users.TestSingleUser#test_not_found_digit[0]", "labels": [{"name": "severity", "value": "minor"}, {"name": "host", "value": "User-pc"}, {"name": "thread", "value": "10832-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "tests.get.test_users"}]}